torch/csrc/jit/interpreter.cpp

```diff
@@ -349,8 +349,8 @@ struct UseList {
 // one instruction plus meta-data
 struct Instruction {
   Operation callback;
-  UseList inputs;
-  ListHandle<int> outputs;
+  UseList inputs{};
```

Comment:
I don't understand the change here. Isn't `UseList` a C++'y type? Then isn't the default constructor sufficient? What makes this different from `Operation`?